### PR TITLE
MGMT-1723 Deploy assisted service and UI using manifests

### PR DESCRIPTION
**Pull request by empovit**

### CODE REVIEW

File: Dockerfile.test-infra [hunk 1]
```diff
@@ -2,9 +2,13 @@ FROM quay.io/ocpmetal/assisted-service:latest AS service
 
 FROM docker.io/centos:8
 
-RUN yum -y install make python3 libvirt-client libvirt-devel gcc unzip wget curl git python3-devel && yum clean all;
+RUN yum -y install make python3 libvirt-client libvirt-devel gcc unzip wget curl git python3-devel yum-utils
```
Comment (eranco74) [inline Dockerfile.test-infra:5]: Why did you remove `yum clean all`?

**Review by eranco74 — COMMENTED** [review]
/lgtm

File: Dockerfile.test-infra (hunk 1 above)
Comment (empovit) [reply Dockerfile.test-infra:5]: I moved it to the end of the installation commands, after the installation of docker-ce and containerd.io. Line 9.

**Review by eranco74 — COMMENTED** [review]
/lgtm
